docs/archived docs/how-to-process-language-models-with-CI-CD.md

```diff
@@ -0,0 +1,86 @@
+# Using botframework-cli tool to process language models for Composer bots
```

Comment:
Can we frame this to Composer's language model processing flow. 

Add a general step description on top of this will be helpful.